tests/conftest.py

```diff
@@ -2334,12 +2334,7 @@ def empty_ge_cloud_data_context_config(
 
 
 @pytest.fixture
-@mock.patch(
-    "great_expectations.data_context.store.DatasourceStore.list_keys",
-    return_value=[],
-)
 def empty_cloud_data_context(
-    mock_list_keys: mock.MagicMock,  # Avoid making a call to Cloud backend during datasource instantiation
```

Comment:
No longer making external requests with datasource init so we remove the mock.

Reply:
Love it